src/main/scala/ai/privado/languageEngine/python/tagger/collection/CollectionTagger.scala

```diff
@@ -64,12 +64,20 @@ class CollectionTagger(cpg: Cpg, sourceRuleInfos: List[RuleInfo]) extends ForkJo
         val parameters =
           collectionMethod.parameter.where(_.name(sourceRule.combinedRulePattern)).whereNot(_.code("self")).l
         val matchingLocals = collectionMethod.local.code(sourceRule.combinedRulePattern).l
-
-        if (parameters.isEmpty && matchingLocals.isEmpty) {
+        val matchingLiterals = collectionMethod
+          .call("(?:get).*")
+          .argument
+          .isLiteral
+          .code("\"(" + sourceRule.combinedRulePattern + ")\"")
```

Comment:
@hiteshbedre along with `"` also include `'` as single and double quotes both can be used for literal in python

Reply:
Handled here: https://github.com/Privado-Inc/privado-core/pull/379/commits/9b7f2b2eeebe69aad75f39683db4f09ad679e32e